### PR TITLE
changed Port field to Optional

### PR DESCRIPTION
**Pull request by Abhishekism9450**
Making Port attribute to Optional.

### CODE REVIEW

File: client/karbon/karbon_private_registry_structs.go [hunk 1]
```diff
@@ -12,9 +12,9 @@ type PrivateRegistryIntentInput struct {
 	Name     *string `json:"name" mapstructure:"name, omitempty"`
 	Cert     *string `json:"cert" mapstructure:"cert, omitempty"`
```
Comment (siddharth-kulshrestha) [inline client/karbon/karbon_private_registry_structs.go]: Please remove space between comma and omitempty.

Comment (siddharth-kulshrestha) [reply client/karbon/karbon_private_registry_structs.go]: In the whole file.

**Review by siddharth-kulshrestha — CHANGES_REQUESTED** [review]

File: client/karbon/karbon_private_registry_structs.go (hunk 1 above)
Comment (Abhishekism9450) [reply client/karbon/karbon_private_registry_structs.go]: Done.

**Review by siddharth-kulshrestha — APPROVED** [review]

**Comment by kingsleyck** [comment]
Thanks all, incredible turnaround.

**Pull request merged**